wallet/src/de/schildbach/wallet/Constants.java

```diff
@@ -228,9 +228,6 @@
 
     public static final int SDK_DEPRECATED_BELOW = Build.VERSION_CODES.JELLY_BEAN;
 
-    public static final boolean BUG_OPENSSL_HEARTBLEED = Build.VERSION.SDK_INT == Build.VERSION_CODES.JELLY_BEAN
-            && Build.VERSION.RELEASE.startsWith("4.1.1");
```

Comment:
We don't need this anymore since the app only supports Android 19 (4.4, Kitkat) and above.

I have fond memories of the heartbleed bug fixes!

Reply:
xD fond memories of bug fixing? :D